npm-shrinkwrap.json

```diff
@@ -1810,8 +1810,9 @@
       }
     },
     "emberjs-build": {
-      "version": "0.5.3",
-      "from": "emberjs-build@0.5.3",
+      "version": "0.5.2",
```

Comment:
Can you revert this back to the current values?  emberjs-build was updated to fix bugs in minification